app/v2/controllers/requestParsers/validators/AmendUkPropertyPeriodSummaryValidator.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.requestParsers.validators
+
+import com.google.inject.Inject
+import config.AppConfig
+import v2.controllers.requestParsers.validators.validations._
+import v2.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
+import v2.models.request.amendUkPropertyPeriodSummary.{AmendUkPropertyPeriodSummaryRawData, AmendUkPropertyPeriodSummaryRequestBody}
+import v2.models.request.common.ukFhlProperty.UkFhlProperty
+import v2.models.request.common.ukNonFhlProperty.UkNonFhlProperty
+
+import javax.inject.Singleton
+
+@Singleton
+class AmendUkPropertyPeriodSummaryValidator @Inject()(appConfig: AppConfig) extends Validator[AmendUkPropertyPeriodSummaryRawData] {
+
+  private lazy val minTaxYear = appConfig.minimumTaxV2Uk
+  private val validationSet   = List(parameterFormatValidation, bodyFormatValidation, bodyFieldFormatValidation)
+
+  private def parameterFormatValidation: AmendUkPropertyPeriodSummaryRawData => List[List[MtdError]] =
+    (data: AmendUkPropertyPeriodSummaryRawData) => {
+      List(
+        NinoValidation.validate(data.nino),
+        TaxYearValidation.validate(minTaxYear, data.taxYear),
+        BusinessIdValidation.validate(data.businessId),
+        SubmissionIdValidation.validate(data.submissionId)
```

Comment:
Please add a test case for bad submissionId: validation spec still passes if this is removed.